tensorboard/plugins/hparams/list_session_groups.py

```diff
@@ -114,6 +114,9 @@ def _session_groups_from_tags(self):
         )
         session_groups = self._filter(session_groups, filters)
         self._sort(session_groups, extractors)
+
+        _reduce_to_hparams_to_include(session_groups, self._request.col_params)
```

Comment:
It is very possible I just do not understand the flow here. However, this seems like we are simply removing the hparams from the response instead of not fetching them at all. Are we sure this will increase performance? Have we performed any performance tests with this change?

Reply:
This was discussed offline to help in my understanding.